### DISCUSSION TITLE
Footnotes

### DISCUSSION THREAD

**Discussion by fabriziodevincenzi** [chrismwilliams/astro-theme-cactus#108] [Ideas]
Hi @chrismwilliams , I really appreciate your work and this specific theme. One feature I really like is the table of contents on the right sidebar, a very useful option for in-depth content. Content for which it would also be nice to have a footnote option. I paste here a link to three examples of how they could be implemented https://codepen.io/cadars/pen/dyXaGdy

I hope it's a welcome idea. Best wishes.
Fabrizio

**Comment by chrismwilliams** [comment]
Hi @fabriziodevincenzi, thanks for the suggestion, and I would like to see something like this added. Once I've updated to astro 3.0 I'll take a look, as I'm sure there's a markdown plugin I've come across that adds this.

**Reply by fabriziodevincenzi** [reply]
Thanks for your attention. I hope there will be a similar implementation :)

**Reply by fabriziodevincenzi** [reply]
Just saw the integration! Beautiful solution.

**Reply by chrismwilliams** [reply]
Thanks for your help, turns out remark-gfm included it all along
